inclusive-design/wecount.inclusivedesign.ca
# fix: the search container no longer overlaps the nav menu on ipad pro (resolves #264)

## Description

To resolve #264.

Fix the issue that the search container no longer overlaps the nav menu on ipad pro.

This pull request is based off the other pull request https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/233 that should be merged first.

## Steps to test

1. Open a web page;
2. Switch to the responsive design and use ipad pro;
3. The search container should not overlap with the "views" element on the nav menu;
4. Test the focus/hover style of all elements on the header nav bar;
5. The focus/hover border of the "views" element should not overlap with the search container.

**Expected behavior:** 
Described above in the test steps.
